server/scripts/import-tests/pg_dump_2021_05_test_data.sql

```diff
@@ -0,0 +1,145 @@
+--
```

Comment:
What is this file doing? Is it a backup? Do you think it should be committed or not?

Reply:
It's what is being used to populate the test database! See `scripts/import-tests/populate-test-data.js`